### ISSUE TITLE
Missing ACORN files

### ISSUE THREAD

**Issue by ocehugo** [aodn/content#474]
Hi @scosoli & @yashahoo,

I was just doing a check on ACORN and detected some further missing ACORN files missing here:

1. IMOS_ACORN_V_20091116T060000Z_TURQ_FV01_1-hour-avg.nc
2. IMOS_ACORN_RV_20191014T020000Z_DONG_FV01_radial.nc
3. IMOS_ACORN_RV_20181031T043000Z_RRK_FV00_radial.nc
4. IMOS_ACORN_RV_20190903T061000Z_GUI_FV00_radial.nc 
5. IMOS_ACORN_RV_20191019T220000Z_CWI_FV00_radial.nc
6. IMOS_ACORN_RV_20200327T044000Z_CWI_FV00_radial.nc
7. IMOS_ACORN_RV_20200327T121000Z_CWI_FV00_radial.nc 
8. IMOS_ACORN_RV_20200907T045500Z_NNB_FV00_radial.nc 

The above files were submitted, but were blocked and landed on our error directory. However, no follow up file with corrections followed, so the above are not published. Some of them are quite old (TURQ/DONG) still from https://github.com/aodn/content/issues/459.

Let me know if you are able to resubmit valid versions of the above.

**Comment by ocehugo** [comment]
Dear @scosoli & @yashahoo,

over Easter, we got a plethora of RT files that failed to be ingested and are missing in our database. See [here](https://github.com/aodn/content/files/6308142/hfradar.log) for blamed/missing files.

**Comment by scosoli** [comment]
Yes I am aware of this issue as the mail was flooded with error messages. 
two things happened: 
1, the RT stream was not updated to process radials from GUI in the new file format
2, the main system drive was full and this prevented files to be handled properly.

Will take care of that in the next few days
